src/h_testkit/factoryboy.py

```diff
@@ -0,0 +1,14 @@
+import tests  # pylint:disable=import-error
+
+
+def set_factoryboy_sqlalchemy_session(db_session):
+    from factory.alchemy import SQLAlchemyModelFactory
```

Comment:
I want the cookiecutter to add h-testkit as a test requirement to all projects indiscriminately. It just seems handy to have h-testkit ready to be used in all our projects, even if some of them don't actually use it yet.

This means that projects that don't depend on SQLAlchemy need to be able to have h-testkit installed without crashing. If imports like this one were top-level in this file then they would crash at pytest startup because SQLAlchemy isn't installed. So put the imports inside the actual functions, that way as long as a project doesn't try to use this function it won't try to import SQLAlchemy.